x-pack/plugins/siem/public/pages/overview/summary.tsx

```diff
@@ -39,10 +39,10 @@ export const Summary = pure(() => (
               </EuiLink>
             ),
             siemSolution: (
-              <EuiLink href="https://www.elastic.co/solutions/security-analytics" target="blank">
+              <EuiLink href="https://www.elastic.co/solutions/siem" target="blank">
                 <FormattedMessage
                   id="xpack.siem.overview.startedText.siemSolutionLinkText"
-                  defaultMessage="Security Analytics"
+                  defaultMessage="SIEM solution"
```

Comment:
We seem to uppercase `Solution`, at least in the docs: https://www.elastic.co/guide/en/siem/guide/current/index.html